build.rs

```diff
@@ -5,6 +5,7 @@ use protobuf_build::Builder;
 fn main() {
     Builder::new()
         .search_dir_for_protos("proto")
+        .append_include(env!("GOGO_PROTOBUF"))
```

Comment:
I guess this might cause problems when tipb is included by TiKV, due to this variable only exists in tipb's Makefile. How about referring to how kvproto does?

Reply:
They need an `include` which stores all protos needed.

Maybe I should revert the changes to `include`, or find a way to embed the files.

Reply:
Got it, looks like we should stick to how [kvproto](https://github.com/pingcap/kvproto/tree/master/include) does.